src/sentry/static/sentry/app/views/organizationEvents/index.jsx

```diff
@@ -71,7 +68,25 @@ class OrganizationEventsContainer extends React.Component {
       updateParams: this.updateParams,
     };
 
-    this.state = OrganizationEventsContainer.getInitialStateFromRouter(props);
+    const values = OrganizationEventsContainer.getStateFromRouter(props);
+    this.state = {
+      ...values,
+      queryValues: {
+        ...values,
+      },
+    };
+  }
+
+  componentWillReceiveProps(nextProps, nextState) {
```

Comment:
I believe we've stopped using in new code since it's deprecated? (we should probably have this in linter)